09-schema/03-rules.md

```diff
@@ -274,6 +274,136 @@ TypeQLDefine query = TypeQL.define(
 </div>
 
 This rule will make every relation transitive.
+## Optimising Transitive Queries
+
+A common use-case for rules is to infer all transitively reachable concepts from a particular (set of) starting concepts (e.g. finding all teams a particular user is a member of recursively, or all nodes reachable from a given node in a graph, etc.) This section describes how to write efficient transitivity rules when it is known in advance which one of the role-players will be specified. This is often the case and we can formulate our rules to answer such queries more efficiently.
+
+### Simple transitivity
```

Comment:
nitpick: all our section titles seem to consistently use capitalisation of each word, so we should use the same style here

Reply:
Done